### PR TITLE
Vibe 2.0 - css modules migration 6

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4390214512

- [ ] a - https://github.com/mondaycom/monday-ui-react-core/pull/851
  - [ ] 1. Menu
  - [ ] 2. MenuDivider
  - [ ] 3. MenuGridItem
  - [ ] 4. MenuItem
  - [ ] 5. MenuItemButton
  - [ ] 6. MenuTitle
  - [ ] 7. MenuButton
- [ ] b - https://github.com/mondaycom/monday-ui-react-core/pull/855
  - [ ] 8. Toggle
  - [ ] 9. ToggleText
  - [ ] 10. MockToggle
  - [ ] 11. MultiStepIndicator
  - [ ] 12. StepIndicator

### CODE REVIEW

File: src/components/Menu/MenuItem/__tests__/__snapshots__/menuItem-snapshots.jest.js.snap [hunk 1]
```diff
@@ -3,20 +3,20 @@
 exports[`Snapshots renders correctly when disabled 1`] = `
 <li
   aria-current={false}
-  className="monday-style-menu-item monday-style-menu-item--disabled"
+  className="item disabled"
+  data-testid="menu-item_undefined-undefined"
```
Comment (shlomitc) [inline src/components/Menu/MenuItem/__tests__/__snapshots__/menuItem-snapshots.jest.js.snap:7]: shouldn't that be avoided with default test ids?

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: src/components/Menu/MenuItem/__tests__/__snapshots__/menuItem-snapshots.jest.js.snap (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/Menu/MenuItem/__tests__/__snapshots__/menuItem-snapshots.jest.js.snap]: Fixed, check it out, please

File: src/components/Menu/MenuGridItem/__tests__/__snapshots__/menuGridItem-snapshot-tests.jest.js.snap [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 exports[`MenuGridItem renders correctly with a child 1`] = `
 <section
-  className="menu-grid-item--wrapper"
```
Comment (shlomitc) [inline src/components/Menu/MenuGridItem/__tests__/__snapshots__/menuGridItem-snapshot-tests.jest.js.snap:5]: just make sure there's no need for this classname

**Review by shlomitc — APPROVED** [review]
looks good, just a small comment about changed class name in the jest snapshot test of `MenuGridItem`

File: src/components/Menu/MenuGridItem/__tests__/__snapshots__/menuGridItem-snapshot-tests.jest.js.snap (hunk 2 above)
Comment (SergeyRoyt) [reply src/components/Menu/MenuGridItem/__tests__/__snapshots__/menuGridItem-snapshot-tests.jest.js.snap:5]: I'm sure about this one

**Pull request closed without merging**